src/Exceptionless.Core/Jobs/EventPostsJob.cs

```diff
@@ -110,7 +110,9 @@ public class EventPostsJob : QueueJobBase<EventPost> {
 
             _metrics.Gauge(MetricNames.PostsUncompressedSize, payload.LongLength);
             if (uncompressedData.Length > maxEventPostSize) {
-                await Task.WhenAll(CompleteEntryAsync(entry, ep, SystemClock.UtcNow), organizationTask).AnyContext();
+                var org = await organizationTask.AnyContext();
```

Comment:
Seems like we could end up trying to await this task multiple times, but I guess that's ok?

Reply:
It shouldn't be, but even if so I think that is fine.